onnx/checker.cc

```diff
@@ -127,9 +129,33 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& ctx) {
     for (const StringStringEntryProto& entry : tensor.external_data()) {
       if (entry.has_key() && entry.has_value() && entry.key() == "location") {
         has_location = true;
+#ifdef _WIN32
+        std::wstring relative_path = clean_relative_path(utf8str_to_wstring(entry.value()));
+        // Check that normalized relative path starts with ".." on Windows.
```

Comment:
The comment doesn't match the code. You are checking if the string contains "..", not "starts"

Reply:
Updated. Thanks!